python_modules/dagster/dagster/_core/definitions/auto_materialize_policy.py

```diff
@@ -69,30 +69,42 @@ class AutoMaterializePolicy(
 ):
     """An AutoMaterializePolicy specifies how Dagster should attempt to keep an asset up-to-date.
 
-    There are two main kinds of auto-materialize policies: eager and lazy. In essence, an asset with
-    an eager policy will try to immediately materialize after upstream changes, while an asset with
-    a lazy policy will only materialize when necessary in order to satisfy the relevant
-    FreshnessPolicies.
-
-    For an asset / partition of an asset with an _eager_ policy to be auto-materialized, at least
-    one of the following must be true:
-
-    - it is missing
-    - it has a freshness policy that requires more up-to-date data
-    - any of its descendants have a freshness policy that require more up-to-date data
-    - any of its parent assets / partitions have newer data
-
-    For an asset / partition of an asset with a _lazy_ policy to be auto-materialized, at least one
-    of the following must be true:
+    Each policy consists of a set of AutoMaterializeRules, which are used to determine whether an
+    asset or a partition of an asset should or should not be auto-materialized.
+
+    The most common policy is `AutoMaterializePolicy.eager()`, which consists of the following rules:
+
+    - `AutoMaterializeRule.materialize_on_missing()` - Materialize an asset or a partition if it has
+    never been materialized.
+    - `AutoMaterializeRule.materialize_on_parent_updated()` - Materialize an asset or a partition if
+    one of its parents has been updated more recently than it has.
```

Comment:
> one of its parents has been updated more recently than it has.

`has` should be `have` here. Weird ik, but it's referring to `parents` as the subject and not `asset` or `partition`